src/isgbd/model/FlowerVersionDao.java

```diff
@@ -0,0 +1,48 @@
+package isgbd.model;
+
+import com.mysql.jdbc.jdbc2.optional.MysqlDataSource;
+
+import java.sql.*;
+
+/**
+ * Created by Laura on 1/24/2018
+ */
+
+public class FlowerVersionDao {
+    private Connection conn;
+    private FlowerDao flowerDao = new FlowerDao();
+
+    public FlowerVersionDao() {
+        MysqlDataSource dataSource = new MysqlDataSource();
+        dataSource.setUser("root");
+        dataSource.setPassword("root");
+        dataSource.setServerName("localhost");
+        dataSource.setDatabaseName("isgbd3");
+        dataSource.setPortNumber(3306);
+        try {
+            conn = dataSource.getConnection();
+        } catch (SQLException e) {
+            System.out.print("Mysql connection error!");
+        }
+    }
+
+    public void insertFlowerVersions(long id, long buds, long transactionId){
```

Comment:
this method can be called from controller with a flower version created and here just get data from flower version to create statement and execute update